packages/control-property-editor/test/unit/panels/outline/OutlinePanel.test.tsx

```diff
@@ -104,6 +104,7 @@ describe('OutlinePanel', () => {
             scale: 1,
             outline: model,
             filterQuery: filterInitOptions,
+            scenario: 'UI_ADAPTATION',
```

Comment:
Could you please use the `scenario` const from `@sap-ux-private/control-property-editor-common`?

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1422/commits/83134d16a0fff1526e4dddf61c067912f7a5bdf2